crates/service/src/jobs/processor.rs

```diff
@@ -0,0 +1,1237 @@
+//! Background job processor implementation
```

Comment:
Introduced background job logic in order to fetch supported assets and do healthcheck on startup and on specific intervals.

Later this functionality would be used to:
- process metrics
- check order status
- ...